src/main/java/frc/robot/commands/DriveBackwardCommand.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.DrivetrainSubsystem;
+
+import javax.inject.Inject;
+
+
+public class DriveBackwardCommand extends CommandBase {
+    private final DrivetrainSubsystem drivetrainSubsystem;
+    private final Timer timer;
+
+    @Inject
+    public DriveBackwardCommand(DrivetrainSubsystem drivetrainSubsystem) {
+        this.drivetrainSubsystem = drivetrainSubsystem;
+        timer = new Timer();
+
+        addRequirements(drivetrainSubsystem);
+    }
+
+    @Override
+    public void initialize() {
+        timer.start();
+        timer.reset();
+        drivetrainSubsystem.tankDrive(-0.5, -0.5);
```

Comment:
It might be good to make constants for numbers like this so you can name them and make it easier for others to know what they are and what they mean.